files_to_prompt/cli.py

```diff
@@ -114,13 +135,41 @@ def cli(paths, include_hidden, ignore_gitignore, ignore_patterns):
     path/to/file2.py
     ---
     ...
+
+    If the `--xml` flag is provided, the output will be structured as follows:
+
+    Here are some documents for you to reference for your task:
+
+    <documents>
+    <document path="path/to/file1.txt">
+    Contents of file1.txt
+    </document>
+
+    <document path="path/to/file2.txt">
+    Contents of file2.txt
+    </document>
```

Comment:
Just as a head up, this format has drifted or doesn't quite match Claude's latest guidelines. It's a straightforward conversion, though. https://docs.anthropic.com/en/docs/build-with-claude/prompt-engineering/long-context-tips#example-multi-document-structure

Reply:
See d016523c221ea2169d6bab2b266d034a7430f197